packages/@dcl/playground-assets/etc/playground-assets.api.md

```diff
@@ -467,8 +472,17 @@ export type EngineEvent<T extends IEventNames = IEventNames, V = IEvents[T]> = {
     data: Readonly<V>;
 };
 
-// @public (undocumented)
-export type Entity = unknown;
+// Warning: (tsdoc-escape-greater-than) The ">" character should be escaped using a backslash to avoid confusion with an HTML tag
+// Warning: (tsdoc-escape-greater-than) The ">" character should be escaped using a backslash to avoid confusion with an HTML tag
+// Warning: (tsdoc-escape-greater-than) The ">" character should be escaped using a backslash to avoid confusion with an HTML tag
+// Warning: (tsdoc-escape-greater-than) The ">" character should be escaped using a backslash to avoid confusion with an HTML tag
+// Warning: (tsdoc-malformed-html-name) Invalid HTML element: Expecting an HTML name
+// Warning: (tsdoc-malformed-html-name) Invalid HTML element: A space is not allowed here

```

Comment:
Should we mute these warning?

Reply:
We should fix the underlying cause, not mute